packages/scripts/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@zk-email/scripts",
```

Comment:
this should just go in helpers imo?

Reply:
or this dkim selector archive site can be its own site

Reply:
> this should just go in helpers imo?

I thought of it, but helpers is a sdk that other people would be importing to their code. We don't want these scripts to be part of it right.